README.md

```diff
@@ -22,7 +22,7 @@ The Slack Orb comes with a number of included templates to get your started with
 | ------------- | ------------- | ------------- |
 | ![basic_fail_1](./.github/img/basic_fail_1.png)  | basic_fail_1   | Should be used with the "fail" event. |
 | ![success_tagged_deploy_1](./.github/img/success_tagged_deploy_1.png)  | success_tagged_deploy_1   | To be used in the event of a successful deployment job. _see orb usage examples_ |
```

Comment:
If we add the link to one, we should be consistant ;)

Reply:
Thanks @gmemstr must have missed this one when reading through, but I've added another commit to include it